dts/bindings/pwm/nxp,qtmr-pwm.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP QTMR PWM
+
+compatible: "nxp,qtmr-pwm"
+
+include: [pwm-controller.yaml, pinctrl-device.yaml, base.yaml]
+
+properties:
+  prescaler:
+    type: int
+    default: 128
+    description: prescale factor from the qtmr clock. Default to 128
```

Comment:
That's what I'd prefer- On the RT1170 128 is a sane default, but this might not apply for every SOC. If we just make this value `required: true` then users have to determine what is best for their application

Reply:
@PetervdPerk-NXP ping- the `Default to 128` line still needs to be dropped here

Reply:
Done!